runcoms/zlogout

```diff
@@ -8,11 +8,14 @@
 # Execute code only if STDERR is bound to a TTY.
 [[ -o INTERACTIVE && -t 2 ]] && {
 
-# Print the message.
-cat <<-EOF
+# To print a message on logout:
+# uncomment the commented-out lines below, an
```

Comment:
Nit-picking: `uncomment the commented-out lines below, an`

I believe you meant `and`

Reply:
👍